edx_rest_api_client/tests/test_client.py

```diff
@@ -167,21 +78,13 @@ def test_shared_client_credential_jwt_access_token(self):
         """
         body = {'access_token': "my-token", 'expires_in': 1000}
         now = datetime.datetime.utcnow()
-        expected_return = ('my-token', now + datetime.timedelta(seconds=1000))
 
         with freeze_time(now):
             self._mock_auth_api(OAUTH_URL, 200, body=body)
-            actual_return = EdxRestApiClient.get_and_cache_jwt_oauth_access_token(
-                OAUTH_URL, 'client_id', 'client_secret'
-            )
-        self.assertEqual(actual_return, expected_return)
         self.assertEqual(len(responses.calls), 1)
```

Comment:
This fails because the api will only be called if `EdxRestApiClient.get_and_cache_jwt_oauth_access_token` gets called.